### PR TITLE
PDA now tells you that send/rec needs to be turned on for scan to work

### LINKED ISSUE #1896: PDAs with Send/Receive off don't give any sort of heads up when scanning
PDAs that have the send/recv functionality turned off don't give any warning when trying to scan with them.

**To Reproduce**
1. Turn off Send/Receive or have it off by default (e.g. nukeops PDA)
2. Scan in messenger
3. No message is generated

**Expected behavior**
A message about being unable to scan because send/receive is off.

**Additional context**
This is minor but one of those things that could be made clearer to prevent confusion.

### PR DESCRIPTION
**Pull request by Michaellaneous**
[qol][minor]
## About the PR 
Prints a message that reminds the user to turn on send/receive in order to use the scan function in the PDA
closes #1896 

## Why's this needed? 
qol

### CODE REVIEW

File: code/obj/item/device/pda2/base_os.dm [hunk 1]
```diff
@@ -181,14 +181,13 @@
...
-						. += "</ul>"
-
-						if (count == 0 && !page_departments.len)
-							. += "None detected.<br>"
+							if (count == 0 && !page_departments.len)
+								. += "None detected.<br>"
 
 					else if (src.message_mode == 1)
-						. += {"<a href='byond://?src=\ref[src];message_func=clear'>Clear</a> | 
+						. += {"<a href='byond://?src=\ref[src];message_func=clear'>Clear</a> |
```
Comment (ZeWaka) [inline code/obj/item/device/pda2/base_os.dm:190]: this will break spacing

File: code/obj/item/device/pda2/base_os.dm [hunk 2]
```diff
@@ -197,7 +196,7 @@
 						. += "<br>"
 
 					else
-						. += {"<a href='byond://?src=\ref[src];input=mailgroup'>Join/create group</a> | 
+						. += {"<a href='byond://?src=\ref[src];input=mailgroup'>Join/create group</a> |
```
Comment (ZeWaka) [inline code/obj/item/device/pda2/base_os.dm:199]: ditto

Comment (Michaellaneous) [reply code/obj/item/device/pda2/base_os.dm]: fixed. no idea why vsccode is dumb and ignores my settings

File: code/obj/item/device/pda2/base_os.dm [hunk 3]
```diff
@@ -139,20 +139,20 @@
 
 					if (!src.message_mode)
 
-						. += {"<a href='byond://?src=\ref[src];message_func=ringer'>Ringer: [src.message_silent == 1 ? "Off" : "On"]</a> | 
-						<a href='byond://?src=\ref[src];message_func=on'>Send / Receive: [src.message_on == 1 ? "On" : "Off"]</a> | 
+						. += {"<a href='byond://?src=\ref[src];message_func=ringer'>Ringer: [src.message_silent == 1 ? "Off" : "On"]</a> |
+						<a href='byond://?src=\ref[src];message_func=on'>Send / Receive: [src.message_on == 1 ? "On" : "Off"]</a> |
 						<a href='byond://?src=\ref[src];input=tone'>Set Ringtone</a><br>
-						<a href='byond://?src=\ref[src];message_mode=1'>Messages</a> | 
+						<a href='byond://?src=\ref[src];message_mode=1'>Messages</a> |
```
Comment (ZeWaka) [inline code/obj/item/device/pda2/base_os.dm:145]: ditto

Comment (Michaellaneous) [reply code/obj/item/device/pda2/base_os.dm]: fixed

**Review by pali6 — APPROVED** [review]

**Pull request merged**